src/blocks/Slider/schema.ts

```diff
@@ -13,18 +13,15 @@ const LoadableProps = {
     properties: {
         source: {
             type: 'string',
-            enum: ['events', 'blog', 'services'],
+            // add loadable sources here if you use it in your project
+            // enum: ['my-loadable-source-1', 'my-loadable-source-1'],
```

Comment:
it seems to me not very obvious how to add, can we leave an example?

Reply:
yes, I needs a story for slider, created a ticket for it